systemtests/src/test/java/io/enmasse/systemtest/TestBase.java

```diff
@@ -86,4 +89,18 @@ void cleanup() throws Exception {
             throw exception;
         }
     }
+
+    protected void assertDefaultMessaging() {
```

Comment:
remove this. Test base is not for storing assertMethods anymore

Reply:
Where you suggest to store this method afterwards? I don't see point why common assertMethod which is used in most of the tests shouldn't be there, moving it somewhere like testUtils is not right in my opinion.

Reply:
AssertionUtils is a best approach I guess, create static class and store special Asserts there

Reply:
and TestBase is not for storing common methods, we have testbase only for extend junit extension and configuration and basic clean in some case

Reply:
OK, thanks for explanation of current situation, I have create AssertionUtils for these as you suggested. Although I think that we should use as least static classes as possible.